cob_twist_controller/include/cob_twist_controller/augmented_solver.h

```diff
@@ -56,26 +39,41 @@ class augmented_solver
      * default: 150
      *
      */
-    augmented_solver(const KDL::Chain& chain, double eps=0.001, int maxiter=5);
-    ~augmented_solver();
+    AugmentedSolver(const KDL::Chain& chain, double eps=0.001, int maxiter=5);
+    virtual ~AugmentedSolver();
     
     /** CartToJnt for chain using SVD including base and various DampingMethods **/
-    virtual int CartToJnt(const KDL::JntArray& q_in, const KDL::JntArray& last_q_dot, KDL::Twist& v_in, KDL::JntArray& qdot_out, std::vector<float> limits_min, std::vector<float> limits_max, KDL::Frame &base_position, KDL::Frame &chain_base);
+    virtual int CartToJnt(const KDL::JntArray& q_in, const KDL::JntArray& last_q_dot, KDL::Twist& v_in, KDL::JntArray& qdot_out, KDL::Frame &base_position, KDL::Frame &chain_base);
+    virtual int OldCartToJnt(const KDL::JntArray& q_in, const KDL::JntArray& last_q_dot, KDL::Twist& v_in, KDL::JntArray& qdot_out, KDL::Frame &base_position, KDL::Frame &chain_base);
```

Comment:
This is a general comment:
In case the new structure covers the "old" functionality, I vote for dropping the support for the old structure including the functions ```calculate_weighting``` and ```enforce_limits``` which moved to other classes in the new structure!

This would further simplify and clean-up this package...